lms/djangoapps/bulk_email/tasks.py

```diff
@@ -588,13 +567,16 @@ def _send_course_email(entry_id, email_id, to_list, global_email_context, subtas
 
             except SINGLE_EMAIL_FAILURE_ERRORS as exc:
                 # This will fall through and not retry the message.
-                total_recipients_failed += 1
-                log.exception(
-                    f"BulkEmail ==> Status: Failed(SINGLE_EMAIL_FAILURE_ERRORS), Task: {parent_task_id}, SubTask: "
-                    f"{task_id}, EmailId: {email_id}, Recipient num: {recipient_num}/{total_recipients}, Recipient "
-                    f"UserId: {current_recipient['pk']}"
-                )
-                subtask_status.increment(failed=1)
+                if exc.response['Error']['Code'] in [
+                    'InvalidParameterValue', 'MessageRejected', 'MailFromDomainNotVerified'
```

Comment:
It seems like `InvalidParameterValue` may not be an exception that SES recognizes/throws? I'm looking at the list here: https://botocore.amazonaws.com/v1/documentation/api/latest/reference/services/ses.html#client-exceptions. I see `MessageRejected` and `MailFromDomainNotVerified`, but not `InvalidParameterValue`. Should this be included?